### ISSUE TITLE
C3. prefix usage for SWEET reprScimethodology

### ISSUE THREAD

**Issue by zednis** [USGCRP/gcis-ontology#9]
The reprSciMethodology ontology available at http://sweet.jpl.nasa.gov/2.3/reprSciMethodology.owl# may not be called “meth” appropriately at https://data.globalchange.gov/activity/048de6b5-nca3-cddv2-r1-process.ttl  If this is the case, what is the “proper” abbreviation for this namespace?

As an aside, it is not possible to tell whether a term called “methodology” really exists at http://sweet.jpl.nasa.gov/2.3/reprSciMethodology.owl#  Could you look into this and confirm?

**Comment by zednis** [comment]
Is there a technical reason why the prefix "meth" may not be used or is this political?  The prefix used in the source ontology is "meth".  We can use whatever prefix we want in our own serializations - there is no 'proper' prefix.  

We could use "method" or "methods".

Also the usage of meth:Methodology from the example TTL is incorrect.   You are using a class as a property.

```
## Methodology employed
   meth:Methodology "Annual total precipitation amounts were averaged for all gridpoints in the southern Great Plains region. Decadal averages were then calculated. Finally the 1901-1960 average precipitation amount was subtracted from the decadal average amount, and a percentage change in precipitation was calculated."^^xsd:string;
```

This is incorrect.  meth:Methodology is a class.  It should not be used as a property.

One options is to create a property in the gcis ontology and use to reference an instance of a methodology

```
<http://data.globalchange.gov/activity/048de6b5-nca3-cddv2-r1-process>
  a prov:Activity ;
  gcis:methodologyEmployed [ 
    a meth:Methodology ; 
    rdf:value "Annual total precipitation amounts were averaged for all gridpoints in the southern Great Plains region. Decadal averages were then calculated. Finally the 1901-1960 average precipitation amount was subtracted from the decadal average amount, and a percentage change in precipitation was calculated."^^xsd:string 
  ] .
```

I think a better option is to use the methodology as a PROV plan.

In PROV plans are used on qualified associations between agents and activities.  You could represent the software used as an agent and create a qualified association between the software and the agent.

```

<http://data.globalchange.gov/software/idl-8-0>
  a gcis:Software, prov:SoftwareAgent ;
  rdfs:label "IDL (version 8.0)"^^xsd:string .

<http://data.globalchange.gov/activity/048de6b5-nca3-cddv2-r1-process>
  a prov:Activity ;
  prov:wasAssociatedWith <http://data.globalchange.gov/software/idl-8-0> ;
  prov:qualifiedAssociation [
    a prov:QualifiedAssociation ;
    prov:agent  <http://data.globalchange.gov/software/idl-8-0> ;
    prov:hadPlan [
      a meth:Methodology ;
      rdf:value "Annual total precipitation amounts were averaged for all gridpoints in the southern Great Plains region. Decadal averages were then calculated. Finally the 1901-1960 average precipitation amount was subtracted from the decadal average amount, and a percentage change in precipitation was calculated."^^xsd:string ;
    ];
  ] ;
  .
```

I think this is the best solution and it makes deeper use of PROV.  

As for the second request on this ticket - I don't understand what you are asking for.  Could you rephrase?

**Comment by justgo129** [comment]
I'm fine with leaving "meth" as is given your reasoning.

With regard to the general issue of using classes as properties, I agree 100% and the GCIS team definitely noticed it during the course of writing the turtle.  Usage of some of those terms arose because no better term (or use of a term) was readily available for illustrating the relationships.

**Comment by zednis** [comment]
We need to find and remove all cases where a class is being used as a property.

**Comment by justgo129** [comment]
Yes, but I'll defer to my colleagues on this.  If we remove all cases where a class is being used as a property, we'll lose much information in the Turtle until other terms are either created or modified.  I'm uncomfortable with the idea of having to go back incrementally to add new/revised (but to be used correctly) terms.  In short, I'm fine with removing such cases of classes being used as properties but not until we have suitable replacements for at least some of them in order to facilitate the process of rectifying.

**Comment by justgo129** [comment]
Regarding the second request of the ticket, I'll rephrase.  Could you confirm that the term "Methodology" is one of the terms listed in http://sweet.jpl.nasa.gov/2.3/reprSciMethodology.owl# ?  I see it there but am unclear whether it is actually a separate entity.  Although a moot point given the nature of this discussion, I am still curious about it.

**Comment by zednis** [comment]
@justgo129 from http://sweet.jpl.nasa.gov/2.3/reprSciMethodology.owl# on line 22

```
<owl:Class rdf:about="#Methodology">
<rdfs:subClassOf rdf:resource="http://sweet.jpl.nasa.gov/2.3/humanResearch.owl#ExperimentActivity"/>
</owl:Class>
```

**Comment by zednis** [comment]
I am going to create a separate ticket regarding updating the instance data to no longer uses classes as properties.

edit - the new ticket for the classes as properties issue is #12

**Comment by aulenbac** [comment]
Do we have a requirement to use SWEET? Quoting the website "SWEET 2.3 is highly modular with 6000 concepts in 200 separate ontologies."

**Comment by justgo129** [comment]
(Apologies if this is a duplicate.  I answered the thread using e-mail but I don't see that comment here.)

 There is no requirement to use SWEET.  Feel free to search for other terms.  The rationale behind using SWEET is that it contains terms about experimentation, scientific method, and broad topics of study which I haven't seen in other ontologies or in wikipedia/dbpedia, or for which the definitions in wikipedia don't coincide with how we are trying to use them in GCIS.  In this specific use case (reprScimethodology), the wikipedia/dbpedia entry for "Methodology"

http://en.wikipedia.org/wiki/Methodology

may not exactly match what we're trying to capture as I don't consider documentation of the steps performed to create an image as "a branch of knowledge."

**Comment by justgo129** [comment]
Having explored this issue further, I believe that we should look for an alternate term.  This is because:
1) The location denoted for meth:Methodology no longer resolves.
"@"prefix meth: http://sweet.jpl.nasa.gov/2.3/reprSciMethodology.owl# 
2) The owl file suggests that "Methodology" is limited to forms of scientific experimentation such as isotope analysis and carbon dating, not to an approach for image creation.  I've copied and pasted the first lines of the .owl file at:
https://gist.github.com/justgo129/5838c83ca582a3e1ddde

So, let's find a different term or create something specific to our use.

**Comment by justgo129** [comment]
How about using or adapting gcis:wasCreatedBy or gcis:wasCompiledBy in-lieu of meth:Methodology?

**Comment by zednis** [comment]
we could drop `meth:Methodology` and just use `prov:Plan`.

```
prov:hadPlan [
      a prov:Plan ;
      rdf:value "Annual total precipitation amounts were averaged for all gridpoints in the southern Great Plains region. Decadal averages were then calculated. Finally the 1901-1960 average precipitation amount was subtracted from the decadal average amount, and a percentage change in precipitation was calculated."^^xsd:string ;
    ];
```

**Comment by justgo129** [comment]
+1

**Comment by justgo129** [comment]
By the way, we could also remove mentions of the "meth" namespace from GCIS since this was the sole instance in which it was being used.

**Comment by justgo129** [comment]
Addressed in:
https://github.com/USGCRP/gcis/pull/200.  Once merged, we can close https://github.com/USGCRP/gcis-ontology/issues/9.

**Comment by justgo129** [comment]
Closed #9 on account of merged https://github.com/USGCRP/gcis/pull/200.
